src/scheduler.ts

```diff
@@ -23,6 +23,10 @@ export class Scheduler {
         this._subject = new Subject();
     }
 
+    public get performance(): Window['performance'] {
+        return this._performance;
+    }
```

Comment:
Maybe it's better to just expose the `now()` method instead of the full `performance` object.

```suggestion
    public now(): number {
        return this._performance.now();
    }
```